### PR TITLE
doc: add minutes for meeting 8 April 2021

### PR DESCRIPTION
**Pull request by mhdawson**
Signed-off-by: Michael Dawson <mdawson@devrus.com>

### CODE REVIEW

File: meetings/2021-04-08.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
+# Node.js Technical Steering Committee (TSC) Meeting 2021-04-08
+
+## Links
+
+* **Recording**: <https://youtu.be/F7WascwpoLU>
+* ***GitHub Issue**: <https://github.com/nodejs/TSC/issues/1009>
```
Comment (richardlau) [inline meetings/2021-04-08.md:6]:
```suggestion
* **GitHub Issue**: <https://github.com/nodejs/TSC/issues/1009>
```

**Review by gireeshpunathil — APPROVED** [review]

**Review by danbev — APPROVED** [review]

**Review by BethGriggs — APPROVED** [review]

**Comment by DeeDeeG** [comment]
I think there is an indentation/nesting syntax issue. A couple of extra spaces before the `*`. Some PRs' main bullet points are nested as if they are discussion bullet points -- interspersed with discussion of other (to my eyes unrelated) PRs.

File: meetings/2021-04-08.md [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+  * Robert: Maybe streams and http teams run the relevant benchmarks. We were lucky Matteo is doing the benchmarks for streams etc.
+  * Ruben: It’s often primordials that cause the performance regressions. It’s not obvious that primordials would have a performance decrease. It has happened so often that I fear we might have reduced our perf in multiple cases without noticing.
+  * Matteo: I agree. People with knowledge of subsystems have not been tagged into primordial PRs.
+  * Ruben: It’s not only streams. Lots of other performance sensitive areas. util.inspect is an example. I do not see any benefit to using primordials. You will have lots of external code vulnerable.
+  * Michael: Sounds like a separate issue.
+  * Ruben: I’ll open an issue about that.
+  * Michael: Maybe we need a higher bar for primordials changes.
+  * lib: refactor Timeout, ImmediateList, TimersList to class [#37408](https://github.com/nodejs/node/pull/37408)
+    * On agenda for visibility, going to remove from agenda
+
```
Comment (Trott) [inline meetings/2021-04-08.md:73]:
``` suggestion
* http: significant performance regression on master [#37937](https://github.com/nodejs/node/issues/37937)
  * mostly resolved, so no discussion needed.
  * If you see code that might affect perf or http, (event emitter, ..) please run performance
    regression
  * Robert: Pre-release task?
  * Matteo: There’s already a lot on the releasers checklists.
  * Michael: People need to run the benchmarks when they see code that might be affected.
  * Ruben: Benchmarks need to be revised. It takes weeks to run them. We can’t have the releasers do that.
  * Robert: Maybe streams and http teams run the relevant benchmarks. We were lucky Matteo is doing the benchmarks for streams etc.
  * Ruben: It’s often primordials that cause the performance regressions. It’s not obvious that primordials would have a performance decrease. It has happened so often that I fear we might have reduced our perf in multiple cases without noticing.
  * Matteo: I agree. People with knowledge of subsystems have not been tagged into primordial PRs.
  * Ruben: It’s not only streams. Lots of other performance sensitive areas. util.inspect is an example. I do not see any benefit to using primordials. You will have lots of external code vulnerable.
  * Michael: Sounds like a separate issue.
  * Ruben: I’ll open an issue about that.
  * Michael: Maybe we need a higher bar for primordials changes.
* lib: refactor Timeout, ImmediateList, TimersList to class [#37408](https://github.com/nodejs/node/pull/37408)
  * On agenda for visibility, going to remove from agenda

```

File: meetings/2021-04-08.md [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+  * Rich: Maybe we just leave it alone.
+  * Robert: Do we have a way to discourage usage other than legacy/deprecation?
+  * Nikita: Perhaps we can emphasize the promise APIs in the docs.
+  * Tobias: We’ve done that in the past.
+  * Michael: +1
+  * Robert: We need to encourage usage of the Promise-based APIs.
+  * Michael: Docs are a good first step.
+  * Tobias: Ecosystem adoption of promises in 3rd party packages is important. Many packages still don’t support promises themselves, and mixing them with promises is troublesome.
+  * Michael: Leave it on the agenda for next week. Let’s see if we can get a champion or champions to take this on and figure it out.)
+* http: significant performance regression on master [#37937](https://github.com/nodejs/node/issues/37937)
```
Comment (Trott) [inline meetings/2021-04-08.md:56]:
```suggestion

* http: significant performance regression on master [#37937](https://github.com/nodejs/node/issues/37937)
```

File: meetings/2021-04-08.md [hunk 4]
```diff
@@ -0,0 +1,133 @@
...
+  * Michael: People need to run the benchmarks when they see code that might be affected.
+  * Ruben: Benchmarks need to be revised. It takes weeks to run them. We can’t have the releasers do that.
+  * Robert: Maybe streams and http teams run the relevant benchmarks. We were lucky Matteo is doing the benchmarks for streams etc.
+  * Ruben: It’s often primordials that cause the performance regressions. It’s not obvious that primordials would have a performance decrease. It has happened so often that I fear we might have reduced our perf in multiple cases without noticing.
+  * Matteo: I agree. People with knowledge of subsystems have not been tagged into primordial PRs.
+  * Ruben: It’s not only streams. Lots of other performance sensitive areas. util.inspect is an example. I do not see any benefit to using primordials. You will have lots of external code vulnerable.
+  * Michael: Sounds like a separate issue.
+  * Ruben: I’ll open an issue about that.
+  * Michael: Maybe we need a higher bar for primordials changes.
+* lib: refactor Timeout, ImmediateList, TimersList to class [#37408](https://github.com/nodejs/node/pull/37408)
```
Comment (Trott) [inline meetings/2021-04-08.md:72]:
```suggestion

* lib: refactor Timeout, ImmediateList, TimersList to class [#37408](https://github.com/nodejs/node/pull/37408)
```

**Comment by Trott** [comment]
> I think there is an indentation/nesting syntax issue.

Thanks. I think I fixed it.

**Review by Trott — APPROVED** [review]

**Pull request merged**